src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -803,6 +803,13 @@ def _exec(
             run_tracker.allow_generator_types = allow_generator_output
             run_tracker.end_run(line_run_id, result=output)
             aggregation_inputs = self._extract_aggregation_inputs(nodes_outputs)
+        except KeyboardInterrupt as ex:
```

Comment:
if should_use_async is false, the KeyboardInterrupt would be ignored?

Reply:
Good catch, should raise out side if